gh_pages/_source/tesseract_setup_wizard_doc.rst

```diff
@@ -0,0 +1,275 @@
+**********************
+Tesseract Setup Wizard
```

Comment:
Same comment; this probably belongs in `tesseract_ignition` rather than this repo